python/src/ExpectationSimulationAlgorithm_doc.i.in

```diff
@@ -0,0 +1,177 @@
+%feature("docstring") OT::ExpectationSimulationAlgorithm
+"Expectation computation using sampling.
+
+Incremental Monte Carlo sampling algorithm to estimate the mean :math:`\Expect{\vect{X}}` of a random vector
+:math:`\vect{X}`.
+
+Parameters
+----------
+X : :class:`~openturns.RandomVector`
+    The random vector to study.
+
+Notes
+-----
+The algorithm can operate on a multivariate random vector :math:`\vect{X}`.
+
+There are 3 mathematical stopping criteria available:
+
+- through an operator on the coefficient of variation
+- through an operator on the standard deviation
+- on the maximum standard deviation per component
+
+The criterion on the coefficient of variation is defined using either:
+
+  - The maximum: :math:`\max_i \frac{\sigma_i}{|\mu_i|} \leq \max_{cov}`
+  - The norm-1: :math:`\frac{1}{d} \sum_1^{d} \frac{\sigma_i}{|\mu_i|} \leq \max_{cov}`
+  - The norm-2: :math:`\sqrt{\frac{1}{d} \sum_1^{d} \left(\frac{\sigma_i}{|\mu_i|}\right)^2} \leq \max_{cov}`
+
+  The type of operator on the coefficient of variation is set using :meth:`setCoefficientOfVariationCriterionType`.
+
+  The default type is set by the `ExpectationSimulationAlgorithm-DefaultCoefficientOfVariationCriterionType`
+  :class:`~openturns.ResourceMap` key.
+
```

Comment:
Please mentio the possible keys (NONE, MAX, NORM1, NORM2)

Reply:
It's explained already in the accessors.